demos/testnet_app/src/testnet_app_assets/src/index.js

```diff
@@ -0,0 +1,35 @@
+import { Actor, HttpAgent } from '@dfinity/agent';
+import { AuthClient } from "@dfinity/auth-client";
+import { idlFactory as testnet_app_idl, canisterId as testnet_app_id } from 'dfx-generated/testnet_app';
+
+
+const testnetII = "https://rdmx6-jaaaa-aaaaa-aaadq-cai.identity.dfinity.network/"
```

Comment:
maybe add a note in the README that this is hardcoded? do we want to be able to run this anywhere else?

_EDIT:_ I see this is an autofill. Worth adding a note about this here.

Reply:
I moved the binding into the event handler. Hopefully that makes it easier to see its small scope.

I also added a comment describing what we're doing.